tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -496,13 +515,16 @@ public function theAdminHasCreatedUser(
 		string $user,
 		string $password,
 		string $email,
-		string $displayName
+		string $displayName,
+		string $userAsAdmin = null
 	): void {
+		$adminUsername = $userAsAdmin ? $this->featureContext->getActualUsername($userAsAdmin) : $this->featureContext->getAdminUsername();
+		$adminPassword = $userAsAdmin ? $this->featureContext->getPasswordForUser($userAsAdmin) : $this->featureContext->getAdminPassword();
 		$response = GraphHelper::createUser(
 			$this->featureContext->getBaseUrl(),
 			$this->featureContext->getStepLineRef(),
-			$this->featureContext->getAdminUsername(),
-			$this->featureContext->getAdminPassword(),
+			$adminUsername,
+			$adminPassword,
```

Comment:
Example is here https://github.com/owncloud/ocis/blob/23d1c5a008cc374fbe2ae8c499f973974e9c6cb4/tests/acceptance/features/bootstrap/GraphContext.php#L153-L162